src/etc/inc/interfaces.inc

```diff
@@ -3079,6 +3125,10 @@ function DHCP6_Config_File_Basic($interface, $wancfg, $wanif, $id = 0)
                 $dhcp6cconf .= "  prefix ::/{$preflen} infinity;\n";
             }
             foreach (link_interface_to_track6($interface) as $friendly => $lancfg) {
+                $intf_stats = legacy_interface_stats(get_real_interface($friendly));
```

Comment:
this can go then, DHCP6_Config_File_Advanced() have the check and interface_dhcpv6_prepare() does it before calling consistently.

Reply:
yes, not needed, I've removed it from my source. At the moment I'm just trying to keep my source up to date with yours. If you give me 10 minutes I'll send an update that should include all your changes and my changes too. ;)